python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -537,7 +550,8 @@ class RandomForestRegressor(Base):
         return predicted_result
 
     def predict(self, X, predict_model="GPU",
-                algo='BATCH_TREE_REORG', convert_dtype=True):
+                algo='BATCH_TREE_REORG', convert_dtype=True,
```

Comment:
It makes sense to change the default `algo` to `AUTO`. If `BATCH_TREE_REORG` is used, FIL will always try to use dense storage (unless `storage_type='SPARSE'`), as only dense storage supports this algorithm.

Reply:
@Salonijain27 @canonizer quick comment that had totally slipped my mind, can we make the parameter values lower case by default in the python functions here for consistency?

Reply:
Sure, feel free to make the default values lowercase.

Btw, lowercase values are also documented in the Python API for FIL: https://github.com/rapidsai/cuml/blob/branch-0.13/python/cuml/fil/fil.pyx#L428

Reply:
will update the files to make the parameter values lower case by default